addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10313,7 +10313,11 @@ msgctxt "#20244"
 msgid "Android Apps"
 msgstr ""
 
-#empty strings from id 20245 to 20249
+msgctxt "#20245"
```

Comment:
Oooh so it triggers the settings app! Then it is starting to make sense, I
thought it was hooks to actually change them :)

Then it might be sane, our program listing is just directory vfs anyway.

On Mon, Apr 6, 2015 at 9:25 PM, Chris Browet <notifications@github.com>
wrote:

> In addons/resource.language.en_gb/resources/strings.po
> <https://github.com/xbmc/xbmc/pull/6888#discussion_r27822214>:
>
> > @@ -10313,7 +10313,11 @@ msgctxt "#20244"
> >  msgid "Android Apps"
> >  msgstr ""
> >
> > -#empty strings from id 20245 to 20249
> > +msgctxt "#20245"
>
> In this instance, settings are just like Android apps. They are shortcuts
> to the one and only droid settings app, each shortcut opening the proper
> page/tab.
> AFAIK, there is just no way for a normal app like us to access the
> underlying values.
>
> An alternative would be to mix them with the normal apps vfs (as it's
> basically the same thing), but would be more cumbersome, imo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/6888/files#r27822214>.
>